src/BloomBrowserUI/bookEdit/js/editViewFrame.js

```diff
@@ -137,10 +110,154 @@ function initializeReaderSetupDialog() {
     document.getElementById('settings_frame').contentWindow.postMessage(fontMsg, '*');
 }
 
-
 /**
  * Called by C# after the setup data has been saved, following Save click.
  */
 function closeSetupDialog() {
     $('#synphonyConfig').dialog("close");
+}
+// "endregion" ReaderSetup dialog
+
+// "region" Add Page dialog
+function FindOrCreateAddPageDiv(templatesJSON, descriptionLabel, blankPreviewMsg) {
```

Comment:
Renamed to CreateAddPageDiv (also the reader setup one that was its pattern).
Actually the iframe is appended inside of the config div, so the 'div' part is accurate.